python/modem-firmware-1.3+/device_credentials_installer.py

```diff
@@ -320,7 +304,9 @@ def wait_for_prompt(val1=b'gateway:# ', val2=None, timeout=15, store=None):
         elif store != None and (store in line or str(store) in str(line)):
             output = line
 
-    lf_done = b'\n' in line
+    if b'\n' not in line:
+        sys.stdout.write('\n')
```

Comment:
the `lf_done` global was a pain in my side when attempting to modularize, so I got rid of it and replaced it with this.

@plskeggs Can you confirm this has the same intended effect? I didn't notice any adverse effects when testing, at least

Reply:
I think it does, but it's easier to check by running the script and confirming that there are not extra LFs (blank lines) when the output from the modem is displayed.

Reply:
Just double-checked and it looks good to me, so I will resolve